compliance-monitor/monitor.py

```diff
@@ -299,6 +372,21 @@ async def get_reports(
         return db_get_reports(cur, subject, limit, skip)
 
 
+@app.get("/reports/{report_uuid}")
+async def get_report(
+    account: Annotated[tuple[str, str], Depends(auth)],
+    conn: Annotated[connection, Depends(get_conn)],
+    report_uuid: str,
+):
+    with conn.cursor() as cur:
+        specs = db_get_report(cur, report_uuid)
+        if not specs:
+            raise HTTPException(status_code=404)
+        spec = specs[0]
+        check_role(account, spec['subject'], ROLES['read_any'])
+    return Response(content=json.dumps(spec, indent=2), media_type="application/json")
```

Comment:
```suggestion
        return Response(content=json.dumps(spec, indent=2), media_type="application/json")
```

(or unindent the previous line, because `check_role` doesn't seem to belong to the `with` either)

Reply:
You could be right. The cursor is only used for the db query. Then I will unindent everything but that query.